src/scss/styles-12-upstream.scss

```diff
@@ -0,0 +1,17 @@
+$mainblue: #152935;
+$maingrey: #F5F5F5;
+$brighterblue: #3D70B2;
+$accentgrey: #dadada;
```

Comment:
Are all of these vars used in the handlebars?

Reply:
removed